tests/integration_test/run_int_test_locally.sh

```diff
@@ -37,6 +42,9 @@ bash ${SCRIPT_DIR}/integration_test.sh \
         "master" \
         "" \
         "github" \
+        "dev" \  # we use `dev` as the default branch in secondary-analysis-deploy repo, not `master`
```

Comment:
This in-line comment is causing some issues when I try to run this test locally. Can you move it into the comment block above?

Reply:
Good catch. I moved it locally but forgot to push. Pushed now :(